.github/workflows/complex-code-spotter.yml

```diff
@@ -273,46 +273,6 @@ jobs:
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
 
-  deny:
-
-    needs: [code-coverage, weighted-code-coverage]
-
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v4
-
-    - name: Check dependencies changes
-      uses: dorny/paths-filter@v2
-      id: changes
-      with:
-        filters: |
-          cargo:
-            - 'Cargo.toml'
-            - 'Cargo.lock'
-
-    - name: Install Rust stable
-      if: steps.changes.outputs.cargo == 'true'
-      uses: dtolnay/rust-toolchain@stable
-      with:
-        toolchain: stable
-
-    - name: Install cargo-deny
-      if: steps.changes.outputs.cargo == 'true'
-      env:
-        DENY_LINK: https://github.com/EmbarkStudios/cargo-deny/releases/download
-        DENY_VERSION: 0.13.7
-      run: |
-        curl -L "$DENY_LINK/$DENY_VERSION/cargo-deny-$DENY_VERSION-x86_64-unknown-linux-musl.tar.gz" |
-        tar xz -C $HOME/.cargo/bin --strip-components 1
-
-    - name: Run cargo-deny
-      if: steps.changes.outputs.cargo == 'true'
-      run: |
-        cargo deny init
-        cargo deny check bans
-        # cargo deny check licenses
-
```

Comment:
We do not have to remove this check, but replace its content looking at `ci-generate`